### ISSUE TITLE
Show energy instead of power in aggregated view table

### ISSUE THREAD

**Issue by corradio** [electricitymap/electricitymap-contrib#4428]
The aggregated view currently shows GW (power). It represents the average power of a zone during the year.
I think it would be more intuitive to show GWh or TWh (energy). It would represent the total energy of a zone.
Also, this is a value which is usually communicated about by experts and by journalists. So far, I've never seen the power value being used in the media.

Mathematically, we could simply take the power value and multiply by the number of hours (it would therefore be robust to missing values).

<img width="420" alt="image" src="https://user-images.githubusercontent.com/1655848/183297361-0f692922-8fc2-4128-8479-b36003a347cd.png">

**Comment by VIKTORVAV99** [comment]
I kind of suggested the same thing in https://github.com/electricitymap/electricitymap-contrib/issues/3938#issuecomment-1171088990 but it probably a good idea for it to have it's own issue.

This should also be changed on the backend so it returns the total energy instead of average power so we can avoid doing these calculations in the frontend.

As for the calculations themselves I think it would be a good idea to include a disclaimer if there is a period with missing information so people don't take it as the absolute truth when we know there is data missing even if it might not change anything. (similar to how the estimated data label work).

**Comment by Kongkille** [comment]
Yes this would be a nice follow up. I am guessing we'd want to do the same for the 'carbon emissions' tab? To show total CO2?

<img width="438" alt="image" src="https://user-images.githubusercontent.com/45588799/183846378-8cc461b9-0b40-4329-89a9-41c6dde8ce04.png">

**Comment by VIKTORVAV99** [comment]
Yeah that seems only logical. Perhaps we could update the current ones to show a value per hour as well since all other values are per hour?

**Comment by Kongkille** [comment]
> Yeah that seems only logical. Perhaps we could update the current ones to show a value per hour as well since all other values are per hour?

Yes it's a bit curious that it's t/min. I am not sure if there's any specific reasoning for it

**Comment by VIKTORVAV99** [comment]
@Kongkille @madsnedergaard @tonypls is this something we could include in the rewrite?
Would be nice to get it added as it would be more clear for the historical views.

**Comment by madsnedergaard** [comment]
While it would be really nice to do, we're already fighting hard to not increase scope of the rewrite project - so let's fix it afterwards :)

**Comment by VIKTORVAV99** [comment]
> While it would be really nice to do, we're already fighting hard to not increase scope of the rewrite project - so let's fix it afterwards :)

Hehe yeah, it feels like it keeps growing ever so slightly with every new PR 😅
Let's put it on the backlog for the v7(+) state files.

**Comment by VIKTORVAV99** [comment]
Now that the rewrite is done how do we feel about starting on this? @madsnedergaard @tonypls

I think it might be good to show a option or toggle to switch between averages (power) and total (energy) as a part of this as well but let's discuss the pros and cons of this first.

**Comment by VIKTORVAV99** [comment]
We are planning on implementing this soon.
The first implementation will be just multiplying the hourly average with the amount of hours in each period. So for daily we would take the hourly average and multiply it with 24 to get the daily energy production.

This can be further improved in the future.
